src/relay/pass/type_infer.cc

```diff
@@ -141,17 +141,7 @@ class TypeInferencer : private ExprFunctor<Type(const Expr&)>,
   Type Unify(const Type& t1, const Type& t2, const NodeRef& expr) {
     // TODO(tqchen, jroesch): propagate span to solver
```

Comment:
Random aside: you can remove the TODO because it's done (since you modify the lines below anyway)